src/test/java/com/wikia/webdriver/common/logging/VelocityWrapper.java

```diff
@@ -31,19 +33,21 @@ private VelocityWrapper() {
     throw new IllegalAccessError("Utility class");
   }
 
-  static String fillLogRow(List<String> classList, String command, String description) {
+  static void fillLogRow(List<LogData> logData, String command, String description) {
     StringBuilder builder = new StringBuilder();
 
     Template t = velocityEngine.getTemplate(LOG_ROW_TEMPLATE_PATH);
     VelocityContext context = new VelocityContext();
 
-    context.put("className", "\"" + classList.stream().collect(Collectors.joining(" ")) + "\"");
+    context.put("className",
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "className" 5 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)